tests/contrib/pytest/test_pytest.py

```diff
@@ -528,6 +528,24 @@ def test_foo():
         for span in spans:
             assert span.get_tag(test.SUITE) == file_name.partition(".py")[0]
 
+    def test_pytest_exception(self):
+        """Test that pytest sets exception information correctly."""
+        py_file = self.testdir.makepyfile(
+            """
+        def test_will_fail():
+            assert 2 == 1
+        """
+        )
+        file_name = os.path.basename(py_file.strpath)
+        self.inline_run("--ddtrace", file_name)
+        spans = self.pop_spans()
+
+        assert len(spans) == 1
+        assert spans[0].get_tag(test.STATUS) == test.Status.FAIL.value
```

Comment:
I'm struggling a bit testing this, as the main pytest tests are being mixed with this inline test